miner/Miner.hs

```diff
@@ -188,15 +174,12 @@ pChainId = optional $ textOption
 
 main :: IO ()
 main = do
-    env@(Env _ _ as) <- Env
-        <$> MWC.createSystemRandom
-        <*> newManager (mkManagerSettings ss Nothing)
-        <*> execParser opts
-    getWork env >>= \case
-        Nothing -> putStrLn "Failed to connect to the given Chainweb Node." >> exitFailure
-        Just wb -> case cmd as of
-            GPU _ -> putStrLn "GPU mining is not yet available."
-            CPU _ -> pPrintNoColor as >> mining (scheme env) env wb >> exitFailure
+    cargs <- execParser opts
+    lopts <- setLogMinLevel (ll cargs) . setLogUseLoc False <$> logOptionsHandle stderr True
```

Comment:
"Be verbose, but set the minimum level to whatever came in from the CLI, and don't bother echoing source line locations."